I think it should not be forced to overwrite assignee

cnlpj opened the pull request:
only if the claimTaskPayload assignee is not set, the current login person is the default assignee

Commit:
I think it should not be forced to overwrite assignee
only if the claimTaskPayload assignee is not set, the current login person is the default assignee

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Activiti/Activiti?pullRequest=3492) <br/>All committers have signed the CLA.

Commit:
I think it should not be forced to overwrite assignee
only if the claimTaskPayload assignee is not set, the current login person is the default assignee
